kymatio/scattering1d/core/scattering1d.py

```diff
@@ -64,6 +64,7 @@ def scattering1d(x, pad, unpad, backend, J, psi1, psi2, phi, pad_left=0,
     irfft = backend.irfft
     cdgmm = backend.cdgmm
     concatenate = backend.concatenate
+    conv = Conv1dFFT(backend)
```

Comment:
Should check the `conv_type` argument, right? If only to error if it's not `'fft'`.

Reply:
The `Conv1DFFT` should be called at the top of the frontend files, methinks. kinda like how the core is called here.

Reply:
Yes the convolution object needs to be tied to the frontend, not to the core. This way it can be involved in the filter preprocessing as discussed above. It'll then be passed into the core and used there, just like the backend.